docs/specs/3d-model-loading/developer-guide.md

```
@@ -1,196 +1,168 @@
 
 ---
 
-## Step 2: Assimp importer
+## Create path (`MeshCreator`)
 
-Add Silk.NET.Assimp. Keep the importer private to the model-loading path (not a public engine service).
-
-On load:
+**Files**: `Engine/Renderer/MeshCreator.cs`, `AssimpModelImporter.cs`, `TextureRelocator.cs`, `MeshWriter.cs`
 
 ```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)